app/lib/emoji.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'singleton'
+
+class Emoji
+  include Singleton
+
+  def initialize
+    data = Oj.load(File.open(File.join(Rails.root, 'node_modules', 'emojione', 'emoji.json')))
```

Comment:
in that case i think the annoying solution is best and using piggo's heretic idea may be the way to go

IDEALLY the separate, versioned emoji json would be the one built into the front end rather than the one from node modules

is that tough?!